libbeat/outputs/logstash/sync.go

```diff
@@ -19,18 +19,21 @@ type client struct {
 	*transport.Client
 	client *v2.SyncClient
 	win    window
+	ttl    <-chan time.Time
```

Comment:
same here, use `time.Ticker` and `ttl.Stop` on close. Otherwise we introduce leaks in case we ever support reloading/reconfiguring outputs.